apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -91,15 +90,22 @@ export class VersionAction extends BaseRushAction {
   }
 
   protected run(): Promise<void> {
+    // try to get the user email
+    const userEmail: string | undefined = GitPolicy.getUserEmail(this.rushConfiguration);
+
+    if (!userEmail) {
+      return Promise.reject(undefined);
+    }
+
     if (!this._bypassPolicy.value) {
-      if (!GitPolicy.check(this.rushConfiguration)) {
-        process.exit(1);
-        return Promise.resolve();
+      if (!GitPolicy.check(this.rushConfiguration, userEmail)) {
+        return Promise.reject(undefined);
```

Comment:
>return Promise.reject(undefined); [](start = 8, length = 33)

FYI Promise.reject() must always be called with an object that implements the `Error` interface